### PR TITLE
allow description to be defined within field block

### PR DESCRIPTION
**Pull request by esjee**
It seems there are no objects to https://github.com/rmosolgo/graphql-ruby/issues/1158, so let's implement it!

### CODE REVIEW

**Comment by Willianvdv** [comment]
@esjee what happens if you define the description in the block _and_ as an argument? Should we refuse the double definition?

**Comment by Willianvdv** [comment]
@esjee lol you already did that 💃 ! Can you spec this behaviour?

File: spec/graphql/schema/field_spec.rb [hunk 1]
```diff
@@ -14,5 +14,17 @@
     it "camelizes the field name" do
       assert_equal 'inspectInput', field.graphql_definition.name
     end
+
+    describe "description in block" do
+      it "will raise if description is defined both in the argument and in the block" do
+        assert_raises RuntimeError do
+          class DescriptionInBlock < Jazz::BaseObject
+            field :should_raise, Jazz::Key, "this should not raise" do
```
Comment (esjee) [inline spec/graphql/schema/field_spec.rb]: This doesn't work! :thinking:

File: spec/support/jazz.rb [hunk 2]
```diff
@@ -59,9 +59,9 @@ def to_graphql
   # A custom field class that supports the `upcase:` option
   class BaseField < GraphQL::Schema::Field
     argument_class BaseArgument
-    def initialize(*args, options, &block)
+    def initialize(*args, **options, &block)
       @upcase = options.delete(:upcase)
-      super(*args, options, &block)
+      super(*args, **options, &block)
```
Comment (Willianvdv) [inline spec/support/jazz.rb]: Nice catch! 🐟

Comment (esjee) [reply spec/support/jazz.rb]: :muscle:

**Comment by rmosolgo** [comment]
👌 !

**Pull request merged**